raiden/tests/scenarios/pfs7_simple_path_rewards.yaml

```diff
@@ -68,7 +68,7 @@ scenario:
             - wait: 10
             - assert_pfs_history: {source: 3, target: 0, request_count: 2}
             - assert_pfs_iou: {source: 3, amount: 200}
-            - assert_pfs_iou: {source: 2, amount: 100}
+            - assert_pfs_iou: {source: 2, amount: 0}
```

Comment:
does that mean that there is no IOU? That might be confusing, it looks like that there is an IOU of amount 0. Maybe just delete that specific assert?